website/tutorial/embassy/index.md

```diff
@@ -215,35 +282,227 @@ components = ["rust-src", "rustfmt", "llvm-tools", "clippy"]
 
 :::tip
 
+We can use the same `rust-toolchain.toml` file for both the **RP2350** and the **STM32U545**, but we need to change the **target triple** depending on which chip we are compiling for. The RP2350 and STM32U545 are both Cortex‑M33 devices, so the correct target is `thumbv8m.main-none-eabi`.
+
+:::
+
+:::tip
+
+<Tabs>
+  <TabItem value="stm32u5" label="STM32 Nucleo-U545RE-Q/Raspberry Pi Pico 2" default>
+Please make sure that you install the Rust ARMv8-M target (thumbv8m.main-none-eabihf).
+
+```bash
+rustup target add thumbv8m.main-none-eabihf
+```
+  </TabItem>
+
+  <TabItem value="rp2350" label="Raspberry Pi Pico" default>
```

Comment:
Add a note that the this step should not be required if the target is placed in the `rust-toolchain.toml` file.